assignments/array-methods.js

```diff
@@ -56,21 +56,32 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach((name)=>{
+  fullName.push(name.first_name,name.last_name)});
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+allCaps.push(runners.map((cap) => {
+  return cap.first_name.toUpperCase()})
```

Comment:
Hong here everything is good but you have to populate the `allCaps` variable with the names using `.push` like you did in the first challenge so you don't return something for every instance of `.map` and just console log the `allCaps` array once